frontend/src/download.js

```diff
@@ -1,37 +1,43 @@
 const FileReceiver = require('./fileReceiver');
 const { notify } = require('./utils');
 const $ = require('jquery');
+require('jquery-circle-progress');
 
 const Raven = window.Raven;
 
 $(document).ready(function() {
   $('#download-progress').hide();
-  $('#send-file').click(() => {
+  $('.send-new').click(() => {
     window.location.replace(`${window.location.origin}`);
   });
+  //initiate progress bar
+  $('#dl-progress').circleProgress({
+    value: 0.0,
+    startAngle: -Math.PI / 2,
+    fill: '#00C8D7',
+    size: 158
+  });
   $('#download-btn').click(download);
   function download() {
     const fileReceiver = new FileReceiver();
     const name = document.createElement('p');
-    const $btn = $('#download-btn');
 
-    fileReceiver.on('progress', percentComplete => {
+    fileReceiver.on('progress', progress => {
       $('#download-page-one').hide();
-      $('.send-new').hide();
       $('#download-progress').show();
+      const percent = progress[0] / progress[1];
       // update progress bar
-      document
-        .querySelector('#progress-bar')
-        .style.setProperty('--progress', percentComplete + '%');
-      $('#progress-text').html(`${percentComplete}%`);
+      $('#dl-progress').circleProgress('value', percent);
+      $('.percent-number').html(`${Math.floor(percent * 100)}`);
+      $('.progress-text').append(
+        ` (${(progress[0] / 1000000).toFixed(2)}MB of ${(progress[1] /
+          1000000).toFixed(2)}MB)`
+      );
```

Comment:
I think this is where the append bug is happening that @abhinadduri was reporting in https://github.com/mozilla/send/pull/191#issuecomment-315141564.